lib/initClient.js

```diff
@@ -0,0 +1,27 @@
+import ApolloClient, { createNetworkInterface } from 'apollo-client'
+
+let apolloClient = null
+
+function createClient (headers) {
+  return new ApolloClient({
+    ssrMode: !process.browser,
+    headers,
+    dataIdFromObject: result => result.id || null,
+    networkInterface: createNetworkInterface({
+      uri: 'http://localhost:3000/graphql',
```

Comment:
window.location.host, window would throw an error, maybe because of SSR. What do you suggest?